nixos-org/webserver.nix

```diff
@@ -154,6 +154,8 @@ in
                 #skins = [ ./wiki-skins ];
                 extraConfig =
                   ''
+                    $wgSiteNotice = "The NixOS wiki is outdated and is being shut down. If you wish to add or improve NixOS documentation, please grab a ticket from the [https://github.com/NixOS/nixpkgs/milestone/8 Move the wiki!] milestone.";
```

Comment:
@simoncarpinter while i do agree with you, i don't see how we can point directly to github issue. if you have any idea how to do this i'm all on improving this.

maybe a next step would be to add a description to miletstone which would explain more.